LondonBritishLibrary/orient/BLorient586.xml

```diff
@@ -54,41 +84,231 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                    <ab type="pricking">Primary pricks are visible, horizontal pricks are visible.</ab>
+                                    <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C</ab>
+                                    <ab type="ruling"> The upper line is written above the ruling,
+                                        the lower line is written above the ruling.</ab>
+                                </layout>
+                                
+                                <layout columns="3" writtenLines="24">
+                                    <locus from="73r" to="195v"/>
+                                    <ab type="pricking">Pricking and ruling are visible.</ab>
+                                    <ab type="pricking">Primary pricks and horizontal pricks are visible.</ab>
+                                    <ab type="ruling" subtype="pattern">1A-1A-1A1A-1A1A/0-0/0-0/C</ab>
+                                    <ab type="ruling"> The upper line is written above the ruling,
+                                        the lower line is written above the ruling.</ab>
+                                </layout>
                             </layoutDesc>
                         </objectDesc>
                         
                         <handDesc>
                             
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is written in a small hand of the <date>18th century</date>
-                                </desc>
-                                
-                                <date notBefore="1700" notAfter="1800"/>
-                                
+                                <desc>Written in a small hand.</desc>
+                                <date notBefore="1700" notAfter="1799"/>
+                                <persName ref="PRS13814MekhaG" role="scribe"/>
                             </handNote>
-                            
                         </handDesc>   
+                        
                         <additions>
-                            
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#1r"/>
+                                    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of 
+                                        <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                                    <q xml:lang="gez">ምዕራፍ፡ ዝማሬ፡ ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName>
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="a2">
+                                    <locus target="#111r"/>
+                                    <desc type="GuestText"></desc>
```

Comment:
Here and below, no indication as to which text it is, with ID? I think the text on f. 111r is the text for mǝhǝllā/mǝhlǝllā service (a part of Mawase't)? It should be in Habtemichael Kidane, in Velat 1966 and in EAE under M.